src/tests/utils/setup.ts

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+import type { StartedPostgreSqlContainer } from '@testcontainers/postgresql';
+import { PostgreSqlContainer } from '@testcontainers/postgresql';
+
+import { insertData } from './insert-data';
+import { initORM, init as initRepositories } from '@repository/index.js';
+import { init as initDomainServices } from '@domain/index.js';
+import config from '@infrastructure/config/index.js';
+import { runTenantMigrations } from '@repository/storage/postgres/migrations/migrate';
+import API from '@presentation/http/http-server.js';
+
+import { beforeAll, afterAll } from 'vitest';
+
+declare global {
+  /* eslint-disable-next-line no-var */
+  var api: API | undefined;
```

Comment:
add some comments pls, why do we need it

Reply:
done